src/main/python/main/ayab/engine/communication_ip.py

```
@@ -0,0 +1,215 @@
+
+import socket
+import ipaddress
+
+import logging
+import pprint
+from time import sleep
+
+# Port for TCP
+remotePort = 12346
+
+class CommunicationIP(Communication):
```

Ruff (0.8.2)

`Communication` may be undefined, or defined from star imports

(F405)